src/main/java/net/sf/jabref/gui/entryeditor/EntryEditorTab.java

```diff
@@ -151,7 +153,12 @@ private void setupPanel(JabRefFrame frame, BasePanel bPanel, boolean addKeyField
                         false);
                 defaultHeight = 0;
             } else {
-                fieldEditor = new TextArea(field, null);
+                String prompt = "";
+                if (field.equals(FieldName.AUTHOR) || field.equals(FieldName.EDITOR)) {
+                     prompt = Localization.lang("Firstname Lastname and Firstname Lastname") + " and others";
```

Comment:
I'd suggest removing the `and` from the localization as well as it is required that it stays `and`, therefore making it easier to translate.